server/src/main/java/nl/inl/blacklab/server/logging/ConsoleLogDatabase.java

```diff
@@ -0,0 +1,62 @@
+package nl.inl.blacklab.server.logging;
+
+import nl.inl.blacklab.requestlogging.LogLevel;
+import nl.inl.blacklab.requestlogging.SearchLogger;
+import nl.inl.blacklab.search.results.SearchResult;
+import nl.inl.blacklab.server.search.BlsCacheEntry;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+/**
+ * ConsoleLogDatabase provides an implementation to LogDatabase, that simply
+ * logs internal blacklab operations to stdout.
+ * ConsoleLogDatabase is a lightweight tracing class to enable debugging of search requests.
+ */
```

Comment:
I think if you just add a note here about trace, this will all be clear. Something like: `Set the log level to trace to see the output from this class.` Thanks!

Reply:
no problem, will do.